src/api/test/unit/bs_request_test.rb

```diff
@@ -152,8 +152,29 @@ def check_user_targets(user, *trues)
     end
   end
 
-  test "request ownership" do
+  def test_request_ownership
     check_user_targets(:Iggy, 10)
     check_user_targets(:adrian, 1, 1000)
   end
+
+  def test_double_entries_are_blocked
```

Comment:
What about a test that ``add_role`` and ``maintenance_incident`` type is allowed? Maybe even in the new test suite?